### PR TITLE
MQTT docs updates and fixes

### PR DESCRIPTION
**Pull request by vordimous**
Live URL for review: https://vordimous.github.io/zilla-docs/next

- clean up outputs
- update link recommendation
- fix double spaces
- update kafka proxy conent and sidebar
- update the mqtt intro configs
- update route exit yaml example
- point to new quickstart folder (#94)
- Remove capabilities (#103)
- Add server to options (#102)
- update the new feature tag
- boostrap fixes
- add mqtt kafka proxy
- add mqtt instructions for the quickstart

### CODE REVIEW

**Comment by vordimous** [comment]
update logos: https://docs.google.com/document/d/1Uy8bNjqiP_BtM-MHPygknx3NZzxnvIAwwEB7fFJSTVY/edit

File: src/README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 home: true
 icon: home
 title: Home
-heroImage: /zilla-rings@2x.png
-heroImageDark: /zilla-rings@2x.png
+heroImage: /zilla-rings.webp
+heroImageDark: /zilla-rings.webp
 heroText: Introduction
-tagline: Zilla is an API Gateway for event-driven architectures. It securely interfaces web apps, IoT clients, and microservices to Apache Kafka® via declaratively defined API endpoints.
+tagline: Zilla is a multi-protocol, edge and service proxy. It abstracts Apache Kafka® for non-native clients, such as browsers and IoT devices, by exposing Kafka topics via user-defined REST, Server-Sent Events (SSE), MQTT, or gRPC API end/entrypoints.
```
Comment (jfallows) [inline src/README.md:8]: Perhaps a matter of personal taste but end/entrypoints doesn't look good to me. I think it looks better to just pick one.

File: src/concepts/kafka-proxies/mqtt-proxy.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+description: This guide will walk through the way Zilla manages MQTT Pub/Sub connections and messages.
+---
+
+# MQTT Kafka Proxy
+
+This guide will walk through the way Zilla manages MQTT Pub/Sub connections and messages.
+
+An MQTT server acts as a broker between publishers and subscribers. This requires a complex protocol to manage the wide range of IoT devices and use cases. By proxying these messages on and off of Kafka with the [mqtt-kafka](../../reference/config/bindings/binding-mqtt-kafka.md) binding, IoT devices can transmit data to a wider range of tech stacks, adapting to more business needs.
+
+Unlike other proxies, Zilla manages the different MQTT topics instead of blindly passing them down to Kafka. This way the Kafka architecture can be optimized for MQTT Pub/Sub. MQTT client subscribers and publishers will communicate with Zilla the same as any broker.
```
Comment (jfallows) [inline src/concepts/kafka-proxies/mqtt-proxy.md:11]: What did you mean by instead of blindly passing them down to Kafka?
Are you saying we don't map MQTT topics to Kafka topics 1-1 ?

File: src/concepts/kafka-proxies/mqtt-proxy.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+```yaml
+topics:
+  messages: mqtt-messages
+  retained: mqtt-retained
+  sessions: mqtt-sessions
+```
+
+### Messages on Kafka
+
+All MQTT messages brokered by Zilla are published on the `messages` Kafka topic. The MQTT message topic becomes the kafka key
```
Comment (jfallows) [inline src/concepts/kafka-proxies/mqtt-proxy.md:67]: This will change later when we support routing to different Kafka topics for different MQTT topic subtrees. (MQTT has a hierarchical topic space)

File: src/tutorials/mqtt/docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      -  |
+        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic mqtt-messages
+        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic mqtt-sessions --config cleanup.policy=compact
+        /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic mqtt-retained --config cleanup.policy=compact
+    depends_on:
+      - kafka
+    init: true
+
+  zilla:
+    image: ghcr.io/aklivity/zilla:develop-SNAPSHOT
```
Comment (jfallows) [inline src/tutorials/mqtt/docker-compose.yaml:37]: This version should change to latest, or does it require a local build?

**Review by jfallows — COMMENTED** [review]
Just some questions. Looks great!

File: src/concepts/kafka-proxies/mqtt-proxy.md (hunk 2 above)
Comment (vordimous) [reply src/concepts/kafka-proxies/mqtt-proxy.md:11]: Yes, when i looked at waterstream and a few others it was common to map an MQTT topic to a Kafka topic. Since Zilla doesn't I wanted to call that out assuming the reader is likely to have seen those other implementations.

File: src/concepts/kafka-proxies/mqtt-proxy.md (hunk 3 above)
Comment (vordimous) [reply src/concepts/kafka-proxies/mqtt-proxy.md:67]: I talked about that briefly with Barnabas. I figured once all of the future routing features are in place, I would update this page anyway and didn't want to worry about the semantics yet. 

Are you ok to leave it like this and updating it later?

**Review by jfallows — APPROVED** [review]

File: src/concepts/kafka-proxies/mqtt-proxy.md (hunk 3 above)
Comment (jfallows) [reply src/concepts/kafka-proxies/mqtt-proxy.md:67]: Yep this was just a heads up for you for later.

File: src/concepts/kafka-proxies/mqtt-proxy.md (hunk 2 above)
Comment (jfallows) [reply src/concepts/kafka-proxies/mqtt-proxy.md:11]: I'd suggest rewording a little to mention 1-1 mapping of MQTT topics to Kafka topics, instead of "blindly".

Comment (vordimous) [reply src/concepts/kafka-proxies/mqtt-proxy.md:11]: I agree your suggestion is a bit more clear. This page is under "concepts" and can have a more casual tone. My goal is to draw more attention to Zilla being different. Once Zilla has more routing features I will revisit this to draw a clearer parallel.
